src/ui/components/accounts/activeAccountCard.tsx

```diff
@@ -23,7 +21,7 @@ interface Props {
 
 export function ActiveAccountCard({
   account,
-  balance,
+  amount,
```

Comment:
Might use a more appropriate name for this prop, e.g. `amountInUsd`?!

Reply:
It seemed to me that the `amount` is enough, because usd are the only amount